transformation-engine/latest/install/msi.md

```diff
@@ -0,0 +1,109 @@
+
+    1. Check the Windows Services in the management console.
+
+    2. Locate the new service called **Document Transformation Engine**, and check that it is **Started**.
+
+> **Note:** Each time a file is transformed in Alfresco Content Services, the `.NET` program starts and Microsoft Office tries to check for a Certificate Revocation List (CRL). Depending on the access that the Document Transformation Engine has to the Internet when transforming a file, this check can delay the operation for up to two minutes, and will therefore, delay transformation of the file. To prevent this, use the Windows server firewall to block internet access for all office binaries.
+
+<!-- (Will be commented back in once 2.4 is released)
+
+## Install the Alfresco Transformation client into Alfresco Content Services
+
+The Alfresco Transformation client is installed as two Alfresco Module Packages (AMP) files into Alfresco Content Services and requires the license to be updated.
+
+Before starting verify that:
+
+* Your Alfresco Content Services server is correctly configured and tested.
+* You have the correct Document Transformation Engine ZIP file for the version of Alfresco Content Services that you are running.
+* You have an updated license file (a `*.lic` file). You can request a license from the [Alfresco Support Portal](http://support.alfresco.com){:target="_blank"}.
+
+1. Stop the Alfresco Content Services server.
+
+2. Open a terminal (Linux) or command line window (Windows).
+
+3. Unzip the `alfresco-document-transformation-engine-2.3.x.zip` file.
+
+4. Copy `alfresco-document-transformation-engine-repo-2.3.x.amp` to the `<ALFRESCO_HOME>/amps` folder, and copy `alfresco-document-transformation-engine-share-2.3.x.amp` to the `<ALFRESCO_HOME>/amps_share` folder.
+
+5. Install the AMP files using the Module Management Tool (MMT).
+
+6. Copy your updated license file into the Alfresco Content Services installation folder.
+
+    Delete all files with the extension `*.installed` in this directory.
+
+7. Start the Alfresco Content Services server.
+
+8. Monitor the Alfresco Content Services log.
+
+    You will see successful log entries about the license installation and the installation of the Alfresco Module Package (depending on the configuration of your log level).
+-->
+
+## Install T-Engine using Docker Compose
+
+To deploy the Document Transformation Engine T-Engine with the Transform Service, you'll need to update your Docker Compose file to include the Document Transformation Engine T-Engine.
+
+> **Note:** While Docker Compose is often used for production deployments, the Docker Compose file provided is recommended for development and test environments only. Customers are expected to adapt this file to their own requirements, if they intend to use Docker Compose to deploy a production environment.
+
+1. Add the Document Transformation Engine T-Engine container to your `docker-compose.yaml` file:
+
+    ```yaml
+    transform-dte-engine:
```

Comment:
this "transform-dte-engine:" needs to match what you are setting for the url in step 2. So either change this to "transform-dte:" or change the url in step 2 to -DlocalTransform.transform-dte.url=http://transform-dte-engine:8090/. I think the former is the way to go as it would follow the naming convention of other t-engines like outlook and the aio t-engine which don't have "-engine" in their service names.